src/backend/main.ts

```diff
@@ -397,6 +398,36 @@ if (!gotTheLock) {
 
     initTrayIcon(mainWindow)
 
+    // hotkey to reload the app
+    globalShortcut.register('CommandOrControl+R', () => {
+      mainWindow.reload()
+    })
+
+    // hotkey to quit the app
+    globalShortcut.register('CommandOrControl+Q', () => {
+      handleExit()
+    })
+
+    // hotkey to open the dev tools
+    globalShortcut.register('CommandOrControl+Shift+I', () => {
+      mainWindow.webContents.openDevTools()
+    })
+
+    // hotkey to open the settings on frontend
+    globalShortcut.register('CommandOrControl+K', () => {
+      sendFrontendMessage('openScreen', '/settings/app/default/general')
+    })
+
+    // hotkey to open the library screen on frontend
+    globalShortcut.register('CommandOrControl+L', () => {
+      sendFrontendMessage('openScreen', '/library')
+    })
+
+    // hotkey to open the downloads screen on frontend
+    globalShortcut.register('CommandOrControl+J', () => {
+      sendFrontendMessage('openScreen', '/download-manager')
+    })
+
```

Comment:
Can we add this maybe in a seperated file?

Reply:
Unfortunately not, I tried moving it to other places but it needs to be done inside the `app.WhenReady` method, see documentation:
https://www.electronjs.org/docs/latest/api/global-shortcut/